src/actions/channel_categories.ts

```diff
@@ -183,6 +179,10 @@ export function moveChannelToCategory(categoryId: string, channelId: string, new
 
         // And remove it from the old category
         const originalCategory = getCategoryInTeamWithChannel(getState(), category.team_id, channelId);
+        const sourceCategories = [{
+            ...category,
+        }, originalCategory];
```

Comment:
This should be moved into the if statement below in case `originalCategory` is somehow null or if it's the same as `category`

Reply:
I don't think this would work because we need `sourceCategories` for the dispatch below.